spec/features/movie/show_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "the movie show page" do
+# Story 2 - Movie Show
+  it "I see the movie attributes and a list of all its actors sorted youngest to oldest with the average age" do
+    studio1 = Studio.create!(name: "Universal Studios", location: "Hollywood")
+    movie1 = studio1.movies.create!(title: "A Movie", creation_year: "2023", genre: "comedy")
+    movie2 = studio1.movies.create!(title: "B Movie", creation_year: "2013", genre: "horror")
+    actor1 = movie1.actors.create!(name: "Sandra Bullock", age: 58)
+    actor2 = movie1.actors.create!(name: "Meryl Streep", age: 73)
+    actor3 = movie1.actors.create!(name: "Emma Watson", age: 33)
+    actor4 = movie2.actors.create!(name: "Person Actress", age: 40)
+    
+    visit "/movies/#{movie1.id}"
+
+    expect(page).to have_content("Movie Title: #{movie1.title}")
+    expect(page).to have_content("Creation Year: #{movie1.creation_year}")
+    expect(page).to have_content("Genre: #{movie1.genre}")
+    expect(actor3.name).to appear_before(actor1.name)
+    expect(actor1.name).to appear_before(actor2.name)
+    expect(page).to have_content("Average Age of Actors: 54.67")
+    expect(page).to_not have_content(movie2.title)
+    expect(page).to_not have_content(actor4.name)
+  end
+
+  # Story 3 - Add an Actor to a Movie
+  it "I do not see any actors listed that are not part of the movie and I see a form to add an actor to the movie" do
+    studio1 = Studio.create!(name: "Universal Studios", location: "Hollywood")
+    movie1 = studio1.movies.create!(title: "A Movie", creation_year: "2023", genre: "comedy")
+    movie2 = studio1.movies.create!(title: "B Movie", creation_year: "2013", genre: "horror")
+    actor1 = movie1.actors.create!(name: "Sandra Bullock", age: 58)
+    actor2 = movie1.actors.create!(name: "Meryl Streep", age: 73)
+    actor4 = movie2.actors.create!(name: "Person Actress", age: 40)
+    
+    visit "/movies/#{movie1.id}"
+    
+    expect(page).to_not have_content("Emma Watson")
+    expect(page).to_not have_content(actor4.name)
+    expect(page).to have_button("Submit")
+    
+    actor3 = movie1.actors.create!(name: "Emma Watson", age: 33)
+    
+    fill_in "add_actor", with: "#{actor3.id}"
+    click_on "Submit"
+    visit "/movies/#{movie1.id}"
+    
+    expect(page).to have_content(actor3.name)
```

Comment:
This test passes because you're creating the actor on line 41 already associated to `movie_1`. If we rework this to only create the actor _but not associate it_ until the form is filled out, the test would not pass because your controller method does not do a `MovieActor.create(...)`.

Reply:
In other words, the test could read like: 
```ruby
expect(page).to_not have_content("Emma Watson")

emma = Actor.create(name: "Emma Watson", age: 33)

fill_in "add_actor", with: emma.id # you did this part just fine
click_on "Submit"
visit "/movies/#{movie_1.id}"

expect(page).to have_content("Emma Watson") # this would not pass with your controller in its current state
```

Reply:
So, we just want to create the actor in the database but not associate it manually, instead that's the functionality we're creating via the form.